BooruSharp/Booru/Template/Gelbooru02.cs

```diff
@@ -35,7 +35,7 @@ protected internal override Search.Post.SearchResult GetPostSearchResult(JToken
                     new Uri("http" + (UsesHttp() ? "" : "s") + "://" + url + "//images/" + elem["directory"].Value<string>() + "/" + elem["image"].Value<string>()),
                     new Uri("http" + (UsesHttp() ? "" : "s") + "://" + url + "//thumbnails/" + elem["directory"].Value<string>() + "/thumbnails_" + elem["image"].Value<string>()),
                     new Uri(_baseUrl + "/index.php?page=post&s=view&id=" + elem["id"].Value<int>()),
-                    GetRating(elem["rating"].Value<string>()[0]),
+                    GetRating(char.ToLower(elem["rating"].Value<string>()[0])),
```

Comment:
I think it's better to also check for upper case letters in the `GetRating(char)` method instead instead of calling `ToLower()` here.

Reply:
Fixed with f7e5e3723e23756b0d059bec5fcc9dfb2b8e3359